hknweb/candidate/templates/candidate/candreq.html

```diff
@@ -90,12 +99,12 @@ <h3>Past Requests</h3>
 
         <!-- Legend for status of challenge request -->
         <p>
-        Confirmed <img src="{% static 'candidate/yes.png'%}" alt="confirmed" style="width: 4%; vertical-align: middle;">
-        Declined <img src="{% static 'candidate/no.png'%}" alt="declined" style="width: 4%; vertical-align: middle;">
-        Waiting for response <img src="{% static 'candidate/maybe.png'%}" alt="in progress" style="width: 4%; vertical-align: middle;">
+        Confirmed <img src="{% static 'candidate/yes.png'%}" alt="confirmed" class="inline-icon">
+        Declined <img src="{% static 'candidate/no.png'%}" alt="declined" class="inline-icon">
+        Waiting for response <img src="{% static 'candidate/maybe.png'%}" alt="in progress" class="inline-icon">
```

Comment:
Could use a div around these to apply `class="inline-icon"` to all three, I believe.

Reply:
I tried this but the style wasn't applied. If you can fix this go ahead, otherwise I'll keep it